src/main/webapp/feed.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Message Feed</title>
+<link rel="stylesheet" href="/css/main.css">
+<link rel="stylesheet" href="/css/user-page.css">
+
+<script>
+
+  // Fetch messages and add them to the page.
+  function fetchMessages(){
+    const url = '/feed';
+    fetch(url).then((response) => {
+      return response.json();
+    }).then((messages) => {
+      const messageContainer = document.getElementById('message-container');
+      if(messages.length == 0){
+       messageContainer.innerHTML = '<p>There are no posts yet.</p>';
+      }
+      else{
+       messageContainer.innerHTML = '';
+      }
+      messages.forEach((message) => {
+       const messageDiv = buildMessageDiv(message);
+       messageContainer.appendChild(messageDiv);
+      });
+    });
+  }
+
+  function buildMessageDiv(message){
+   const usernameDiv = document.createElement('div');
+   usernameDiv.classList.add("left-align");
+   usernameDiv.appendChild(document.createTextNode(message.user));
+
+   const timeDiv = document.createElement('div');
+   timeDiv.classList.add('right-align');
+   timeDiv.appendChild(document.createTextNode(new Date(message.timestamp)));
+
+   const headerDiv = document.createElement('div');
+   headerDiv.classList.add('message-header');
+   headerDiv.appendChild(usernameDiv);
+   headerDiv.appendChild(timeDiv);
+
+   const bodyDiv = document.createElement('div');
+   bodyDiv.classList.add('message-body');
+   bodyDiv.appendChild(document.createTextNode(message.text));
+
+   const messageDiv = document.createElement('div');
+   messageDiv.classList.add("message-div");
+   messageDiv.appendChild(headerDiv);
+   messageDiv.appendChild(bodyDiv);
+
+   return messageDiv;
+  }
+
+  // Fetch data and populate the UI of the page.
+  function buildUI(){
+   fetchMessages();
+  }
+</script>
+</head>
+<body onload="buildUI()">
+ <div id="content">
+  <h1>Message Feed</h1>
+  <hr/>
+  <div id="message-container">Loading...</div>
+ </div>
+</body>
+</html>
```

Comment:
You missed newline here

Reply:
Fixed with commit https://github.com/anhducle98/team-47-codeu/pull/6/commits/6d4c7de777db0d3576f8e83b50dbc731912276db